torch/autograd/variable.py

```diff
@@ -741,6 +741,27 @@ def __len__(self):
     def __iter__(self):
         return iter(map(lambda i: self[i], range(self.size(0))))
 
+    def __eq__(self, other):
+        return NotImplemented
```

Comment:
At some point in the not too distant future we should implement these even if they return outputs that are marked as "not differentiable"